### DISCUSSION TITLE
Hello, I just added jq into the x-cmd pkg

### DISCUSSION THREAD

**Discussion by edwinjhlee** [stedolan/jq#3051] [Show and tell]
Hello, friends in jq community.

I just add jq into the x-cmd pkg. Now jq users can simply 'x jq' to use jq.

And we record some scriptplay demos for jq:
- https://www.x-cmd.com/pkg/jq#jq
- https://www.x-cmd.com/1min/jq#jq

I apologize in advance if it's not appropriate to raise this topic using a GitHub issue.

**Comment by itchyny** [comment]
Thanks for letting us know.

**Reply by edwinjhlee** [reply]
I think I recognize you [itchyny](https://github.com/itchyny). You are author of gojq. 
If you don't mind,  I will add gojq, gojo and bed into x-cmd/pkg tomorrow. 

There is a bunch of go/rust tools in x-cmd pkg. I think you might probably like them too.

**Reply by edwinjhlee** [reply]
BTW, will jq add yml support just like gojq did ?

**Reply by itchyny** [reply]
See #467. Personally I think that's unlikely, yq is there and available widely.
